docs/public-networks/concepts/transactions/pool.md

```diff
@@ -11,22 +11,23 @@ tags:
-  pool to use.
-- [`--tx-pool-layer-max-capacity`](../../reference/cli/options.md#tx-pool-layer-max-capacity) option
-  to specify the maximum memory capacity of the transaction pool.
-- [`--tx-pool-price-bump`](../../reference/cli/options.md#tx-pool-price-bump) option to specify the
-  price bump percentage to replace an existing transaction.
-- [`--tx-pool-priority-senders`](../../reference/cli/options.md#tx-pool-priority-senders)
-  option to specify sender addresses to prioritize in the transaction pool.
-- [`newPendingTransactions`](../../how-to/use-besu-api/rpc-pubsub.md#pending-transactions) and
-  [`droppedPendingTransactions`](../../how-to/use-besu-api/rpc-pubsub.md#dropped-transactions) RPC
-  subscriptions to notify you of transactions added to and dropped from the transaction pool.
-
+Transaction pools are categorized into the following two types: 
+
+* [Layered](#layered-transaction-pool) - Recommended for public blockchain networks.
+* [Sequenced](#sequenced-transaction-pool) - Recommended for private blockchain networks. 
+  
+Options and methods for configuring and monitoring the transaction pool include the following:
+
+|                                                                                              | Description                                                                      | Transaction pool type|
+|----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|----------------------|
+| [`txpool_besuTransactions`](../../reference/api/index.md#txpool_besutransactions)            | API method to list transactions in the transaction pool.                         | Both                 |
+| [`--tx-pool`](../../reference/cli/options.md#tx-pool)                                        | Option to specify the type of transaction pool to use.                           | Both                 |
+| [`--tx-pool-layer-max-capacity`](../../reference/cli/options.md#tx-pool-layer-max-capacity)  | Option to specify the maximum memory capacity of the transaction pool.           | Layered only         |
+| [`--tx-pool-price-bump`](../../reference/cli/options.md#tx-pool-price-bump)                  | Option to specify the price bump percentage to replace an existing transaction.  | Both                 |
+| [`--tx-pool-priority-senders`](../../reference/cli/options.md#tx-pool-priority-senders)      | Option to specify sender addresses to prioritize in the transaction pool.        | Both                 |
+| [`newPendingTransactions`](../../how-to/use-besu-api/rpc-pubsub.md#pending-transactions)     | RPC subscription to notify you of transactions added to the transaction pool.    | Both                 |
+| [`droppedPendingTransactions`](../../how-to/use-besu-api/rpc-pubsub.md#dropped-transactions) | RPC subscription to notify you of transactions dropped from the transaction pool.| Both                 |
```

Comment:
I think it's a little confusing to include only some tx pool options and methods in a table, without specifying why these are more special than the other options – maybe we should either list *all* tx pool related options or none.

Reply:
okay. This was a list in the docs previously, so I just converted it to a table. I will just add the --tx-pool options that are missing.